docs/1.0/customization/extensions.md

```diff
@@ -34,3 +35,50 @@ $environment->addExtension(new EmojiExtension();
 $converter = new CommonMarkConverter([], $environment);
 echo $converter->convertToHtml('Hello! :wave:');
 ```
+
+## Included Extensions
+
+Starting in v1.3, this library includes several extensions to support Github-Flavored Markdown.  You can manually add the GFM extension to your environment like this:
+
+```php
+$environment = Environment::createCommonMarkExtension();
+$environment->addExtension(new GithubFlavoredMarkdownExtension();
```

Comment:
Missing a `)`:
```php
$environment->addExtension(new GithubFlavoredMarkdownExtension());
```

Reply:
Good catch!  This has been fixed in 34829b26